src/main/java/service/StorageService.java

```diff
@@ -0,0 +1,13 @@
+package service;
+
+import java.util.List;
+
+public interface StorageService {
+    void add(String fruit, Integer count);
+
+    void set(String fruit, Integer count);
+
+    void subtract(String fruit, Integer count);
+
+    List<String[]> getBalance();

```

Comment:
Dao layer must contain only CRUD methods. Any other logic must be placed in service classes. For example: subtract and add operations must be in your operation handlers